gamemodes/terrortown/entities/weapons/weapon_zm_carry.lua

```diff
@@ -65,40 +65,44 @@ SWEP.PrevOwner = nil
 
 -- ConVar syncing
 if SERVER then
-
 	---
 	-- @realm server
-	local cvAllowRagCarry = CreateConVar("ttt_ragdoll_carrying", "1", {FCVAR_NOTIFY, FCVAR_ARCHIVE})
+	local cvAllowRagCarry =
+		CreateConVar("ttt_ragdoll_carrying", "1", { FCVAR_NOTIFY, FCVAR_ARCHIVE })
 
 	---
 	-- @realm server
-	local cvPropForce = CreateConVar("ttt_prop_carrying_force", "60000", {FCVAR_NOTIFY, FCVAR_ARCHIVE})
+	local cvPropForce =
+		CreateConVar("ttt_prop_carrying_force", "60000", { FCVAR_NOTIFY, FCVAR_ARCHIVE })
 
 	---
 	-- @realm server
-	local cvPropThrow = CreateConVar("ttt_prop_throwing", "1", {FCVAR_NOTIFY, FCVAR_ARCHIVE})
+	local cvPropThrow = CreateConVar("ttt_prop_throwing", "1", { FCVAR_NOTIFY, FCVAR_ARCHIVE })
 
 	---
 	-- @realm server
-	local cvAllowRagPin = CreateConVar("ttt_ragdoll_pinning", "1", {FCVAR_NOTIFY, FCVAR_ARCHIVE})
+	local cvAllowRagPin = CreateConVar("ttt_ragdoll_pinning", "1", { FCVAR_NOTIFY, FCVAR_ARCHIVE })
 
 	---
 	-- @realm server
-	local cvAllowRagPinInno = CreateConVar("ttt_ragdoll_pinning_innocents", "0", {FCVAR_NOTIFY, FCVAR_ARCHIVE})
+	local cvAllowRagPinInno =
+		CreateConVar("ttt_ragdoll_pinning_innocents", "0", { FCVAR_NOTIFY, FCVAR_ARCHIVE })
```

Comment:
I mean it is even differnt in the same file. Is this because of the line length? If so, I think this should be handled differently. Because your argument for this is that wordwrap is bad. But is this really any better?

Reply:
wordwrap and long lines are bad for coding:
- wordwrap kills consistent pointer and scrollmovement
- long lines either hide content or lead to wordwrap, or even worse it forces you to scroll horizontally (see this exact github code window)

Reply:
I've used wordwrap for at least 10 years of coding and I had never problems when scrolling or cursor placements. Agree on the second point though.

Maybe chars is the problem them. It is probably due to old machines having an 80 charactter line limit. I checked our codebase (at least 10 files or so) and the longest line we had was 117 characters. Most are way shorter. I think we could solve most of my issues by increasing the line length to 120. Is there a solid reason we chose 100? That's not even half my screen.

Reply:
github choose this as the cutoff point for horizontal scrolling. if you want to look at how our code looks with 120 chars limit take a look at the second commit in this pr.

Reply:
![image](https://github.com/TTT-2/TTT2/assets/13639408/47c4783b-c0aa-4962-b725-f21a6ac7bee0)

At least 111 chars looks fine on my FullHD screen

Reply:
![above_100_chars_github_scroll](https://github.com/TTT-2/TTT2/assets/16392835/d6230fde-8594-431d-925d-79f296f582c5)

Reply:
Also this discussion is not about 100 chars or 120 chars. If you prefer 100, that's ok with me. I'm mostly annoyed that this autoformatter adds so many different styles of indentation instead of sticking to a single one. I get that this is done automatic. But This makes the code not good looking to me if I scroll through a file and see 2 or 3 different styles of convar initialization directly after one another

Reply:
> ![above_100_chars_github_scroll](https://private-user-images.githubusercontent.com/16392835/292561191-d6230fde-8594-431d-925d-79f296f582c5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDMyNjgxMDMsIm5iZiI6MTcwMzI2NzgwMywicGF0aCI6Ii8xNjM5MjgzNS8yOTI1NjExOTEtZDYyMzBmZGUtODU5NC00MzFkLTkyNWQtNzlmMjk2ZjU4MmM1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjIyVDE3NTY0M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRmMWVmN2NiNWYwOTBiNmU0MDM5MDQ4YmI2ZjQyYzdhMzAwYjQ3YmE3ODA2YWZmYzk2YzEyMTI5MjVlNTMwMjgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.sfAhxvRarOyxZ-Wfl9qYcWI7x0eaATYda95jbi2PtLc)

I might be blind, but I don't get what you're trying to show to me here

Reply:
> > ![above_100_chars_github_scroll](https://private-user-images.githubusercontent.com/16392835/292561191-d6230fde-8594-431d-925d-79f296f582c5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDMyNjgxMDMsIm5iZiI6MTcwMzI2NzgwMywicGF0aCI6Ii8xNjM5MjgzNS8yOTI1NjExOTEtZDYyMzBmZGUtODU5NC00MzFkLTkyNWQtNzlmMjk2ZjU4MmM1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjIyVDE3NTY0M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRmMWVmN2NiNWYwOTBiNmU0MDM5MDQ4YmI2ZjQyYzdhMzAwYjQ3YmE3ODA2YWZmYzk2YzEyMTI5MjVlNTMwMjgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.sfAhxvRarOyxZ-Wfl9qYcWI7x0eaATYda95jbi2PtLc)
> 
> I might be blind, but I don't get what you're trying to show to me here

There is a scrollbar and that line is 103 chars long (i think)